src/atap_widgets/conversation.py

```diff
@@ -334,6 +334,29 @@ def get_topic_recurrence(
 
         return recurrence
 
+    @staticmethod
+    def get_conceptual_recurrence_rate(similarity: pd.DataFrame) -> float:
```

Comment:
Should this raise a ValueError or at least give a warning for conversations with < 2 utterances? Input validation.